server/src/test/java/org/opensearch/gateway/ReplicaShardBatchAllocatorTests.java

```diff
@@ -692,6 +745,78 @@ private RoutingAllocation onePrimaryOnNode1And1Replica(AllocationDeciders decide
         );
     }
 
+    private RoutingAllocation twoAssignedPrimaryShardsAnd1UnassignedReplicaPerShard(AllocationDeciders deciders)
```

Comment:
1->one, Can be named as `twoPrimaryAndOneUnAssignedReplica`

Reply:
Done.